index.html

```diff
@@ -1497,14 +1497,24 @@ <h3>Miscellaneous</h3>
 
 
 <section id="locale" class="topic">
-<h2>Local dates, times and formats</h2>
+<h2>Locales, date and time values, and locally affected formats</h2>
 <ul class="summary">
+  <li><a href="#loc_neutral">Working with locale-affected values</a></li>
   <li><a href="#loc_time">Working with time</a></li>
   <li><a href="#loc_names">Working with personal names</a></li>
   <li><a href="#loc_forms">Designing forms</a></li>
   <li><a href="#loc_numbers">Working with numbers</a></li>
 </ul>
 
+<section id="loc_neutral" class="subtopic">
+	<h2>Working with locale-affected values</h2>
+
+<p>Software systems that support languages and cultural preferences are said to be "internationalized". An internationalized system uses APIs to provide language or culturally specific processing, based on user preferences. These user preferences are usually referred to as a "Locale". For more information on general internationalization terminology, see <em>Language Tags and Locale Identifiers</em> [[LTLI]]</p>
```

Comment:
Should "Locale" be lowercase? It is lowercase in the [LTLI](https://www.w3.org/TR/ltli/) document.

Reply:
But not in the title (this quotes most of the title in titlecase)

Reply:
I mean the one in the inside of the quotation marks, not the one in the inside of the `em` (btw, `cite` might be better than `em` here).

Reply:
Fixed in 3226644